src/types/ScreenInfoUiaProvider.cpp

```diff
@@ -31,20 +32,22 @@ SAFEARRAY* BuildIntSafeArray(_In_reads_(length) const int* const data, const int
     return psa;
 }
 
-ScreenInfoUiaProvider::ScreenInfoUiaProvider(_In_ WindowUiaProvider* const pUiaParent) :
+Microsoft::Console::Types::ScreenInfoUiaProvider::ScreenInfoUiaProvider(_In_ Microsoft::Console::Render::IRenderData* pData,

```

Comment:
Do you really have to have the fully-qualified prefix here? You are using the namespace above.

Reply:
(and below)